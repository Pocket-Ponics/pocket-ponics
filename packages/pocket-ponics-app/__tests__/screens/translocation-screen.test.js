import React from 'react'
import { shallow } from 'enzyme'

import TranslocationScreen from '../../screens/translocation-screen'

describe('Translocation Screen', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<TranslocationScreen />
		).html()
		expect(tree).toMatchSnapshot()
	})
})