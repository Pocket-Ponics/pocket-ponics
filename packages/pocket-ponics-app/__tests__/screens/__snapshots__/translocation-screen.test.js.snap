// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Translocation Screen renders correctly 1`] = `"<View style=\\"flex:1\\"><View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:20px;text-align:center\\">Transplant Seedlings</Text><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">Remove the seedling along with the rockwool from the tray. Wrap a moist paper towel around the rockwool. Place the wrapped rockwool into a net pot inside the approriate tier.</Text><Image source=\\"1\\" style=\\"resize-mode:contain;width:100%;flex:1;margin-bottom:20px\\"></Image><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Done Transplanting</Text></View></View></View>"`;
